MockerTests/MockerTests.swift

```diff
@@ -258,7 +229,10 @@ final class MockerTests: XCTestCase {
     }
 
     /// It should correctly handle redirect responses.
-    func testRedirectResponse() {
+    func testRedirectResponse() throws {
+        #if os(Linux)
+        throw XCTSkip("The URLSession swift-corelibs-foundation implementation doesn't currently handle redirects directly")
```

Comment:
👌. Perhaps we can even leave a reference to the issue that tracks this here?

Reply:
Here is the code that throws this error https://github.com/apple/swift-corelibs-foundation/blob/main/Sources/FoundationNetworking/URLSession/URLSessionTask.swift#L1037 I haven't found any open issue, seems like `URLSession` hasn't been fully implemented on Linux, also, not sure what's their plans in `swift-corelibs-foundation`